cloudflare.php

```diff
@@ -14,6 +14,15 @@
     exit;
 }
 
+const CF_MIN_PHP_VERSION = '5.3.10';
+const CF_MIN_WP_VERSION = '3.4';
```

Comment:
`const` keyword is supported PHP 5.3+ 

If the user has a lower version it wouldn't matter cause It'll just give a fatal error. We should change it to a normal variable.